workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+
+# Getting Started
+
+Let me give a brief idea of what we are gonna do. We are gonna make a greeting card with maroon-ish back ground with a text written on it saying ***"Merry Cristmas"***.
+Great ! We are done !
+ Nope, just kidding . We are gonna add a christmas tree too. That is, we need a green tree with brown stem, a yellow/golden star on top and tiny red balls on either sides of the tree.
+
+![alt text](https://cloud-4ipo9natc.vercel.app/3merry_christmas.png) 
+
+To break it down a bit more, we have made the leaves and stem out of multiple boxes or rectangles, with tiny circles on either end of the rectangle. And finally a star on top at the centre.
+![alt text](https://cloud-4ipo9natc.vercel.app/1image2.png)
+# Shall we start Coding then ? 
+
+Create a new [repl.it](https://repl.it) project and select **Python** as the language from the drop down menu on the right.
+
+We are gonna make 2 python files . (Writing in single page may make it a bit too complicated to understand)
+1.  ******Figures.py****** - A python file in which we code the functions that we will use to create the boxes/rectangles , the tiny circles and the star.
+2.  ******ChristmasTree.py****** - A python file in which we call the functions of boxes and circles and stars to make a christmas tree. And finally, we will add a greeting on the bottom of the screen as you saw above
+
+# Lets code Figures.py first.
+
+Using the coding space on the left create a file and name it 'Figures.py' (most probably a main.py file will already be created, don't use that).
+Now in the coding window :
+
+First let's import the turtle library :
+```python
+import turtle
+```
+
+Next, let's make a function to draw a rectangle 
+
+```python
+def box1 (turtle, color, x, y, width, height):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+```
+   Was that a bit too much to take in ? No worries,I will  break it down for you. 
+```python
+def box1 (turtle, color, x, y, width, height):
+```
+First we create the function definition with color of the rectangle (color) , postion of the turtle/where to start drawing the rectangle from (x and y and coordinates) and finally width and height of the box/rectangle.
+
+```python
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+```
+We use ******penup()****** and ******pendown()****** function to move the turtle without drawing anything (just like lifting the pen from the paper and keeping back on paper)
+******goto(x,y)****** decides the position of the turtle
+```python
+for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
```

Comment:
Shortform for forward = fd, right= rt, left = lt, backward = back. Maybe mention this if there is a lot of repetition.